src/Pages/CreatorPage.js

```diff
@@ -31,18 +40,17 @@ function CreatorPage() {
   useEffect(getPosts, []);
 
   return (
-    <div>
-      { AuthorDetails !== undefined
-        ? (
-          <AuthorInfo
-            author={AuthorDetails.fields.name !== undefined ? AuthorDetails.fields.name : 'MISSING CREATOR'}
-            bio={AuthorDetails.fields.bio !== undefined ? AuthorDetails.fields.bio : ''}
-            links={AuthorDetails.fields.personal_site !== undefined
-              ? AuthorDetails.fields.personal_site : ''}
-            authorPic={AuthorDetails.fields.image !== undefined
-              ? AuthorDetails.fields.image[0].url : ''}
-          />
-        ) : <p>No such author found!</p> }
+    <div style={styles.root}>
+      {AuthorDetails !== undefined ? (
+        <AuthorInfoCard
+          authorName={AuthorDetails.fields.name}
```

Comment:
The previous version of this code had error checking to handle the cases where any of the fields are missing. Personal sites and images especially could cause problems because we may not have that data from some authors. The previous version is definitely not the cleanest, but we definitely need some error-checking to prevent any app-breaking errors. Can you implement that however you see fit and test it on an empty object in the backend?